wikitextparser/_config.py

```diff
@@ -151,6 +151,20 @@ def regex_pattern(words: _Iterable[str]) -> bytes:
     'BASEPAGENAME',
     'BASEPAGENAMEE',
     'CASCADINGSOURCES',
+    'CURRENTDAY',
+    'CURRENTDAY1',
```

Comment:
I believe it should be ``CURRENTDAY2``, ``CURRENTDAY1`` does not exist (should be treated as template).

Reply:
Oops .. I this time copy/pasted them :D